### ISSUE TITLE
Pytorch使用笔记 | Oliver xu's Blog

### ISSUE THREAD

**Issue by olixu** [olixu/blog-comment#58]
https://blog.oliverxu.cn/2020/08/14/Pytorch%E4%BD%BF%E7%94%A8%E7%AC%94%E8%AE%B0/ 

 Pytorch有非常多的函数，这篇文章长期更新，记录使用到的函数。  参考资源：  Pytorch cheat sheet  Categorical 官方文档：https://pytorch.org/docs/stable/distributions.html CLASS torch.distributions.categorical.Categorical(probs=None, logits
